DirectionsPlayground/DirectionsPlayground/Sources/QueryEditor.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import SwiftUI
+import Combine
+import MapboxDirections
+
+struct QueryEditor: View {
```

Comment:
As a suggestion, names like these sound as if it was a model, not a view. Would it be possible to use names like `QueryEditView`?